code/modules/mob/living/carbon/brain/brain_item.dm

```diff
@@ -53,18 +50,18 @@
 	brainmob << "<span class='notice'>You feel slightly disoriented. That's normal when you're just a [initial(src.name)].</span>"
 	callHook("debrain", list(brainmob))
 
-/obj/item/organ/brain/examine(mob/user) // -- TLE
+/obj/item/organ/internal/brain/examine(mob/user) // -- TLE
 	..(user)
 	if(brainmob && brainmob.client)//if thar be a brain inside... the brain.
 		user << "You can feel the small spark of life still left in this one."
 	else
 		user << "This one seems particularly lifeless. Perhaps it will regain some of its luster later.."
 
-/obj/item/organ/brain/removed(var/mob/living/user)
+/obj/item/organ/internal/brain/remove(var/mob/living/user)

```

Comment:
I'm not sure, but you might need the `special` variable here, so the parent call goes nicely.